pages/linux/lsusb.md

```diff
@@ -0,0 +1,23 @@
+# lsusb
+
+> Display information about USB buses and devices connected to them.
+
+- List all the USB devices available:
+
+`lsusb`
+
+- List the USB hierarchy as a tree:
+
+`lsusb -t`
+
+- List verbose information about USB devices:
+
+`lsusb --verbose`
+
+- List detailed information about a USB device:
+
+`lsusb -D {{device}}`
+
+- List devices with a specified vendor and product id only:
```

Comment:
I don't think you can have multiple devices with the same vendor and product id. In that case, this should be "List a device with a .."

Reply:
I think It will be the same, but I can understand what @agnivade is referring to.

Reply:
Ah that's right. Yes, then I think it is possible to have multiple devices with same vendor and product id.